iphone/Classes/TiRootViewController.m

```diff
@@ -1178,8 +1178,15 @@ - (void)refreshOrientationWithDuration:(id)unused
 #ifdef FORCE_WITH_MODAL
     [self forceRotateToOrientation:target];
 #else
-    [self rotateHostingViewToOrientation:target
-                         fromOrientation:[[UIApplication sharedApplication] statusBarOrientation]];
+    if ([TiUtils isIOS11OrGreater]) {
+      forcingStatusBarOrientation = YES;
+      [[UIApplication sharedApplication] setStatusBarOrientation:target animated:NO];
```

Comment:
This method is deprecated in iOS 9+, we should use a wrapper to use the iOS 10+ API for newer devices.

Reply:
@vijaysingh-axway Can you add the `[TiUtils isLowerThan:@"12.0"]` (or how the method was named) check to only add this workaround for iOS 11?